Nos/Views/SettingsView.swift

```diff
@@ -78,15 +79,18 @@ struct SettingsView: View {
                 .padding(.vertical, 5)
             } header: {
                 VStack(alignment: .leading, spacing: 10) {
-                    Text(.localizable.privateKey)
+                    PlainText(.localizable.privateKey)
```

Comment:
@martindsq @mplorentz Instead of using `SwiftUI.Text` here, I opted to use our `PlainText` since it's a bit shorter, and it should be easy enough to find/replace once we've removed the `Font.swift` funcs. Does that make sense to you, or should we just use `SwiftUI.Text` here?

Reply:
Yeah that makes sense. If Clarity City is no longer our default font then I think we can get rid of our `Text` overload and `PlainText`.